{
  "Header": {
    "Original": "CVE-2019-9866 (An issue was discovered in GitLab Community and Enterprise Edition 11. ...)",
    "Line": 370599,
    "ID": "CVE-2019-9866",
    "Description": "(An issue was discovered in GitLab Community and Enterprise Edition 11. ...)"
  },
  "Annotations": [
    {
      "Original": "- gitlab 11.8.3-1 (bug #925196)",
      "Line": 370600,
      "Type": "package",
      "Package": "gitlab",
      "Kind": "fixed",
      "Version": "11.8.3-1",
      "BugNo": 925196
    },
    {
      "Original": "NOTE: https://about.gitlab.com/2019/03/20/critical-security-release-gitlab-11-dot-8-dot-3-released/",
      "Line": 370601,
      "Type": "NOTE",
      "Description": "https://about.gitlab.com/2019/03/20/critical-security-release-gitlab-11-dot-8-dot-3-released/"
    }
  ]
}